NadekoBot.Core/Modules/Searches/Services/SearchesService.cs

```diff
@@ -360,13 +361,27 @@ public string GetRandomImageUrl(ImageTag tag)
             }
             else if (tag == ImageTag.Goose){
                 return $"https://raw.githubusercontent.com/steggie3/goose-dataset/master/images/goose-mugshot-{_rng.Next(1, max).ToString("0000")}.jpg";
-            }else{
+            }
+            else if (tag == ImageTag.Image){
+                return $"https://picsum.photos/seed/{GetRandomString(16)}/2000";
```

Comment:
16 is arbitrary. lmk if you'd like me to add it as a constant somewhere (I couldn't find them and tbh I didn't bother looking thoroughly since its a huge file 😒)

Reply:
Nah it's fine